source/standalone/workflows/rl_games/train.py

```diff
@@ -22,6 +22,10 @@
 parser.add_argument("--num_envs", type=int, default=None, help="Number of environments to simulate.")
 parser.add_argument("--task", type=str, default=None, help="Name of the task.")
 parser.add_argument("--seed", type=int, default=None, help="Seed used for the environment")
+
+parser.add_argument("--checkpoint", type=str, default=None, help="Path to model checkpoint.")
+parser.add_argument("--sigma", type=str, default=None, help="The policy's initial standard deviation.")
+
```

Comment:
NIT: Maybe better to move this right above max iterations to keep all agent related parameters together?

Reply:
It's kind of a good idea.
should I have to push it now and wait for CodeBuild check again?
( I'm not sure if it needs to be done right away since @Dhoeller19 already approved it yesterday )